lib/msf/core/payload_set.rb

```diff
@@ -97,7 +94,6 @@ def recalculate
 
       # Cache the payload's size
       begin
-        sizes[name] = p.cached_size || p.new.size
       # Don't cache generic payload sizes.
       rescue NoCompatiblePayloadError
       rescue StandardError => e
```

Comment:
Is this empty block and rescue statement needed still?

Reply:
They are not, will remove, nice spot